themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -2,42 +2,51 @@ doLogIn=Entrar
 doRegister=Cadastre-se
 doCancel=Cancelar
 doSubmit=Ok
+doBack=Voltar
 doYes=Sim
 doNo=N\u00E3o
-doContinue=Continue
+doContinue=Continuar
+doIgnore=Ignorar
 doAccept=Aceitar
 doDecline=Rejeitar
-doForgotPassword=Esqueceu sua senha?
+doForgotPassword=Esqueceu a senha?
```

Comment:
Do you really want to change this? If you don't mind, I would just leave as is.

Reply:
Actually I don't mind, I'm leaving as is